vue/src/lib/SignalRAspNetCoreHelper.ts

```diff
@@ -1,15 +1,15 @@
 import AppConsts from './appconst'
 import Util from './util'
-class SignalRAspNetCoreHelper{
-    initSignalR(){
+class SignalRAspNetCoreHelper {
+    initSignalR() {
         var encryptedAuthToken = Util.abp.utils.getCookieValue(AppConsts.authorization.encrptedAuthTokenName);
-
+        let remoteServerUrl = AppConsts.remoteServiceBaseUrl;
         Util.abp.signalr = {
             autoConnect: true,
             connect: undefined,
             hubs: undefined,
             qs: AppConsts.authorization.encrptedAuthTokenName + "=" + encodeURIComponent(encryptedAuthToken),
-            url: AppConsts.remoteServiceBaseUrl + '/signalr'
+            remoteServiceBaseUrl: remoteServerUrl.endsWith('/') ? remoteServerUrl.slice(0, -1) : remoteServerUrl
```

Comment:
Should make this similar to [angular/src/shared/helpers/SignalRAspNetCoreHelper.ts](https://github.com/aspnetboilerplate/module-zero-core-template/blob/master/angular/src/shared/helpers/SignalRAspNetCoreHelper.ts):

```suggestion
            remoteServiceBaseUrl: AppConsts.remoteServiceBaseUrl,
            url: '/signalr'
```

Reply:
Can help to fix in ReactJS project too?

https://github.com/aspnetboilerplate/module-zero-core-template/blob/7bbcbf36fc14a5a9b0dd4b26fd533f43eff9cfd0/reactjs/src/lib/signalRAspNetCoreHelper.ts#L15

Reply:
in reactjs , the `remoteServiceBaseUrl` is `http://localhost:21021`
```bash
GENERATE_SOURCEMAP=false
REACT_APP_APP_BASE_URL="http://localhost:3000"
REACT_APP_REMOTE_SERVICE_BASE_URL="http://localhost:21021"
```
i set ` url: '/signalr'` 


in vue ,the `remoteServiceBaseUrl` is `http://localhost:21021/`
```bash
const URL = process.env.NODE_ENV === 'production' ? 'https://yourdomain/' : 'http://localhost:21021/';
export default URL;
```
i set ` url: 'signalr'`

Reply:
We should remove trailing slash in `remoteServiceBaseUrl` in Vue project instead, to be consistent with other projects and `appBaseUrl` in Vue project:

https://github.com/aspnetboilerplate/module-zero-core-template/blob/20ddd2452e84700448ecc0717d869de35159ba7a/vue/src/lib/appconst.ts#L12-L13

We can do it in **appconst.ts**:

```diff
- remoteServiceBaseUrl:url
+ remoteServiceBaseUrl: url.endsWith('/') ? url.slice(0, -1) : url
```

Reply:
done

Reply:
No, let's do it in **appconst.ts** instead of **SignalRAspNetCoreHelper.ts**.